examples/nlp/token_classification/token_classification.py

```diff
@@ -194,11 +195,7 @@ def create_pipeline(
         if args.use_weighted_loss:
             logging.info(f"Using weighted loss")
             label_freqs = data_layer.dataset.label_frequencies
-            class_weights = nemo.collections.nlp.data.datasets.datasets_utils.data_preprocessing.calc_class_weights(
-                label_freqs
-            )
-
-            logging.info(f"class_weights: {class_weights}")
+            class_weights = calc_class_weights(label_freqs, max_id=-1)
```

Comment:
max_id=-1 is the default, please revert the changes in the script

Reply:
removed.